### PR TITLE
Gan predictions test ratio

### PR DESCRIPTION
**Pull request by ngushchin**

### CODE REVIEW

File: evaluation.py [hunk 1]
```diff
@@ -13,17 +13,18 @@
 ANGLE_IDX = 5
 
 
-def evaluate_model(generator, experiment, test_set, batch_size, batch_num, parametres, device, scaler, step):
+def evaluate_model(generator, experiment, test_set, batch_size, batch_num, parametres, device, scaler, step,
+                   gan_test_ratio):
```
Comment (V3RGANz) [inline evaluation.py]: use just `parameters.gan_test_ratio`

File: evaluation.py [hunk 2]
```diff
@@ -13,17 +13,18 @@
...
     features += DIJET_SYSTEM_FEATURES
     predictions = []
+    generator.eval()
     for _ in tqdm(range(batch_num), desc='evaluation', position=0, leave=True):
         x_noise = torch.randn((batch_size, parametres.gan_noise_size), device=device)
         predictions.append(generator(x_noise).cpu().detach().numpy())
-
     predictions_np = np.concatenate(predictions)
-
+    generator.train()
```
Comment (V3RGANz) [inline evaluation.py]: It is important to switch in eval mode, but switching back to `.train()` usually do inside training function at every iteration beginning. In this case evaluation function do not care about what's going outside it, as well as train function do not care about what's going outside train.

**Review by V3RGANz — CHANGES_REQUESTED** [review]

**Review by V3RGANz — APPROVED** [review]
